### PR TITLE
Content: IBM Cloud Starter App removed

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Given the recent announcement from IBM (https://www.ibm.com/cloud/blog/announcements/deprecation-of-ibm-cloud-starter-kits) that they will no longer provide a starter application for Node-RED, publishing this content will allow us to attach onto that traffic and hopefully get more FF users.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [-] Suitable unit/system level tests have been added and they pass
 - [-] Documentation has been updated

### CODE REVIEW

**Comment by joepavitt** [comment]
Still need to make a quick blog tile too, but putting up so that content can be reviewed

File: src/blog/2023/03/ibmcloud-starter-removed.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: IBM Cloud removes Node-RED starter application
+subtitle: Get up and running with a Starter Application for Node-RED with FlowForge 
+description: IBM Cloud has recently announced that theyw ill no longer be providing their Cloud App Service Starter Kits, including the Node-RED Starter Application. Don't worry though, FlowForge has you covered.
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/ibmcloud-starter-removed.md:4]:
```suggestion
description: IBM Cloud has recently announced that they will no longer be providing their Cloud App Service Starter Kits, including the Node-RED Starter Application. Don't worry though, FlowForge has you covered.
```

File: src/blog/2023/03/ibmcloud-starter-removed.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+date: 2023-03-29
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/ibmcloud-starter-removed.md:5]: Neat trick: Feel free to not use:

```suggestion
date: git created
```

File: src/blog/2023/03/ibmcloud-starter-removed.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+2. **FlowForge Instance Settings:** For a given Instance in FlowForge, click "Settings > Palette". You can then define the npm module name and versions explicitely in the "Installed Modules" section
+
+![Screenshot of FlowForge's "Installed Modules" option in Instance > Settings > Palette](./images/ff-installed-modules.png "Screenshot of FlowForge's 'Installed Modules' option in Instance > Settings > Palette")
+
+It's also easy to setup [Environment Variables](/blog/2023/01/environment-variables-in-node-red/) for Node-RED in FlowForge for when you integrate with external services like APIs too.
+
+## Security
+
+As with IBM Cloud, FlowForge makes it very easy to secure your Node-RED Applications. FlowForge offers three tiers of security options on your Node-RED Instances to secure any exposed HTTP routes on your Node-RED instance, e.g. REST API endpoints or Node-RED Dashboard.
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/ibmcloud-starter-removed.md:44]:
```suggestion
As with IBM Cloud, FlowForge makes it very easy to secure your Node-RED Applications. [FlowForge offers three tiers of security](https://flowforge.com/docs/user/instance-settings/#security) options on your Node-RED Instances to secure any exposed HTTP routes on your Node-RED instance, e.g. REST API endpoints or Node-RED Dashboard.
```

File: src/blog/2023/03/ibmcloud-starter-removed.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+### Manual Import
+
+This will only enable you to import your Flows, not the associated credentials.
+
+1. Export your existing `flows.json` from your IBM Cloud-hosted Node-RED instance by choosing "Export > All Flows > Download" within Node-RED
+2. Create a new Application on FlowForge
+3. Once created, click the Node-RED instance that has been generated within your Application
+4. Click "Settings"
+5. Scroll down and select the "Import Instance" option
+6. Choose your `flows.json` file that you downloaded earlier.
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/ibmcloud-starter-removed.md:68]: You're missing the environment variables? Or doesn't IBM Cloud use those?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2023/03/ibmcloud-starter-removed.md (hunk 4 above)
Comment (joepavitt) [reply src/blog/2023/03/ibmcloud-starter-removed.md:68]: If you're doing this manually, then no easy way of exporting from IBM Cloud that we are aware of (without being able to go in and experiment)

**Comment by joepavitt** [comment]
added tile image:

File: src/blog/2023/03/ibmcloud-starter-removed.md (hunk 2 above)
Comment (knolleary) [reply src/blog/2023/03/ibmcloud-starter-removed.md]: heh - I've ust suggested the opposite given 11ty's docs advise its a compute expensive way of doing it - and it seems needless to require the website to recompute this static value every single time we deploy.

File: src/blog/2023/03/ibmcloud-starter-removed.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+title: IBM Cloud removes Node-RED starter application
```
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/blog/2023/03/ibmcloud-starter-removed.md:2]: fwiw I think this is a stronger title and will get more clicks from social media.

[the remark this answers is not in the document]
Comment (joepavitt) [reply src/blog/2023/03/ibmcloud-starter-removed.md:2]: Not convinced personally, but if we do change it, then we should change the subtitle with it to something like "IBM Cloud has removed their Node-RED starter application - get up and running on FlowForge instead"

[the remark this answers is not in the document]
Comment (joepavitt) [reply src/blog/2023/03/ibmcloud-starter-removed.md:2]: If we don't change the title, then I'd be keen to include the migration phrase into the subtitle instead. Not sure the CTR would be hire, given that most people won't know the start application has been removed in the community, so a standalone migration article (as suggested by title) wouldn't engage many people as they wouldn't know _why_ they would need to migrate

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**